nat-lab/run_local.py

```diff
@@ -82,13 +82,35 @@ def main() -> int:
         verify_setup_correctness()
 
     if not args.nobuild:
-        run_build_command("linux", args)
-        # Run windows tests on WinVM
-        if args.windows:
-            run_build_command("windows", args)
-        # Run nat-lab natively on macOS (TODO: Add windows support)
-        if args.o == "darwin":
-            run_build_command("darwin", args)
+        print("\u001b[33m")
+        print("|=======================================================|")
+        print("| WARNING! Running builds requires atleast 16GBs of RAM |")
+        print("|=======================================================|")
+        print("\u001b[0m")
+        try:
+            run_build_command("linux", args)
+            # Run windows tests on WinVM
+            if args.windows:
+                run_build_command("windows", args)
+            # Run nat-lab natively on macOS (TODO: Add windows support)
```

Comment:
I guess windows is already added, so we can remove the TODO 😄

Reply:
I don't really know what this TODO is about. Don't really know if this works natively on windows tho :smile: Therefore leaving it as it is